lumberjack.go

```diff
@@ -112,6 +112,7 @@ type Logger struct {
 	mu   sync.Mutex
 
 	millCh    chan bool
+	wg        sync.WaitGroup
```

Comment:
this should be a pointer.

Reply:
why? it would not be passed by method, just one property of this struct.

Reply:
@natefinch could you reply this?

Reply:
@jaczhao 
https://golang.org/pkg/sync/#WaitGroup
A WaitGroup must not be copied after first use.

Reply:
so where to init the : wg        sync.WaitGroup

temporary I put it in  file : lumberjack.go

```go

// mill performs post-rotation compression and removal of stale log files,
// starting the mill goroutine if necessary.
func (l *Logger) mill() {
	if l.wg == nil {
		l.wg = new(sync.WaitGroup)
	}

	l.startMill.Do(func() {
		l.wg.Add(1)
		l.millCh = make(chan bool, 1)
		go l.millRun()
	})
	select {
	case l.millCh <- true:
	default:
	}
}


```